File: MyCars/MyCars/Car+CoreDataProperties.m
//
//  Car+CoreDataProperties.m
//  MyCars
//
//  Created by Лолита on 05.03.2021.
//  Copyright © 2021 Ivan Akulov. All rights reserved.
//
//

#import "Car+CoreDataProperties.h"

@implementation Car (CoreDataProperties)

+ (NSFetchRequest<Car *> *)fetchRequest {
	return [NSFetchRequest fetchRequestWithEntityName:@"Car"];
}

@dynamic mark;
@dynamic model;
@dynamic rating;
@dynamic timesDriven;
@dynamic lastStarted;
@dynamic myChoice;
@dynamic imageData;
@dynamic tintColor;

@end

File: MyCars/MyCars/Car+CoreDataProperties.swift
//
//  Car+CoreDataProperties.swift
//  MyCars
//
//  Created by Лолита on 09.03.2021.
//  Copyright © 2021 Ivan Akulov. All rights reserved.
//
//

import Foundation
import CoreData


extension Car {

    @nonobjc public class func fetchRequest() -> NSFetchRequest<Car> {
        return NSFetchRequest<Car>(entityName: "Car")
    }

    @NSManaged public var tintColor: NSObject?
    @NSManaged public var imageData: Data?
    @NSManaged public var lastStarted: Date?
    @NSManaged public var mark: String?
    @NSManaged public var model: String?
    @NSManaged public var myChoice: NSNumber?
    @NSManaged public var rating: NSNumber?
    @NSManaged public var timeDriven: NSNumber?

}
